crates/core/src/db/datastore/locking_tx_datastore/mut_tx.rs

```diff
@@ -1122,24 +1058,26 @@ impl MutTxId {
             committed_state_shared_lock: SharedWriteGuard::downgrade(committed_state_write_lock),
             lock_wait_time: Duration::ZERO,
             timer: Instant::now(),
+            ctx: self.ctx,
```

Comment:
Note, downgrading is the only way to switch contexts. In particular we downgrade a reducer `tx` in order to execute its corresponding incremental update `tx`. This means we should create a new `ctx` here instead of passing the old one.

Reply:
What I should create here? Or you mean I need to pass it on ` pub fn commit_downgrade(self, new_ctx:...)`

Reply:
Ok done with `(commit | rollback)_mut_tx_downgrade(tx, Workload::Subscribe)`

Reply:
We decided on passing the workload to `commit_downgrade`